recipes/clone_prep_clear_config.rb

```diff
@@ -0,0 +1,17 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: clone_prep_clear_config
+#
+# Make a universal forwarder part of a host image.
+#
+# See http://docs.splunk.com/Documentation/Forwarder/latest/Forwarder/Makeauniversalforwarderpartofahostimage
```

Comment:
Change latest to 6.5.2. It has been known that pages change names between versions, and latest is always the latest version (and pages happen to change). 

Possibly also add a comment how this recipe requires Chef 12.6.0 or greater. (That's where the `:before` notification timing was introduced).